onnxscript/function_libs/torch_aten/ops/core.py

```diff
@@ -765,7 +767,7 @@ def aten_bitwise_left_shift(self: TInt, other: TInt) -> TInt:
 @torch_op("aten::bitwise_not")
 def aten_bitwise_not(self: TInt) -> TInt:
     # bitwise_not(Tensor self) -> Tensor
-
+    self = op.Cast(self, to=INT64.dtype)
     return op.BitwiseNot(self)
```

Comment:
Curious what the input type from bloom to this op was?

Reply:
bool

Reply:
If we casted bool to int64, will there be more bits coming out from nowhere?

Reply:
Is bitwise_not only used on bool types in this model? If so, I would create an overload as we can't handle dtypes in a function.

```python
@torch_op("aten::bitwise_not", overload=True)
def aten_bitwise_not_bool(self: BOOL) -> BOOL:
    # bitwise_not(Tensor self) -> Tensor

    return op.Not(self)
```

and then use this `aten_bitwise_not_bool` in place of the original in the dictionary for now.

Reply:
BitwiseNot expects INT anyway, so I thought we can temp cast it inside. The input is from prims_convert_element_type. Casting there makes regression on GPT.

Reply:
I just don't think the behavior will remain the same?

Reply:
Got it now!